components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
We definitely can achieve a similar effect by just writing media queries for things that need to be responsive. It just depends on whether you want a strict grid system or not (which makes it easier to do breakpoints). However, you don't always need a different width for each breakpoint.